### PR TITLE
Allow setMetas in SeoPage to accept float and integer as meta content

### PR DESCRIPTION
**Pull request by slash890**
#112

### CODE REVIEW

File: Tests/Seo/SeoPageTest.php [hunk 1]
```diff
@@ -43,6 +43,18 @@ public function testOverrideMetas()
         $this->assertEquals($expected, $page->getMetas());
     }
 
+    public function testAcceptedMetaContent()
+    {
+        $page = new SeoPage();
+        $page->setMetas(array('property' => array('int' => 123, 'string' => 'i am a string', 'float' => 55.22)));
+
+        $expected = array(
+            'property'   => array('int' => array(123, array()), 'string' => array('i am a string', array()), 'float' => array(55.22, array())),
```
Comment (OskarStark) [inline Tests/Seo/SeoPageTest.php]: to much space before `=>`

Comment (core23) [inline Tests/Seo/SeoPageTest.php]: Please check this @slash890

Comment (slash890) [inline Tests/Seo/SeoPageTest.php]: @core23 I've done this commit 49e583389ef3c97cc7a6c746d87f11c8b9b797cb  to fix space issue before =>, should i rebase everything in one commit or it's fine with 2 commit?

Comment (greg0ire) [inline Tests/Seo/SeoPageTest.php]: One commit would be better.

**Pull request closed without merging**